openshift-install-powervs

```diff
@@ -985,7 +985,17 @@ function variables {
   ALL_NETS=$($CLI_PATH pi nets --json| grep name | cut -f4 -d\" | grep -v "\-pub-net" || true)
   [ -z "$ALL_NETS" ] && error "No private network found"
 
-  ALL_OCP_VERSIONS=$(curl -skL https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/| grep "$RELEASE_VER" | cut -f2 -d '>' | cut -f1 -d '<' | tac)
+  RAW_OCP_VERSIONS=$(curl -skL https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/| grep "$RELEASE_VER" | cut -f2 -d '>' | cut -f1 -d '<' | tac)
+  ALL_OCP_VERSIONS=""
+  for item in $RAW_OCP_VERSIONS
+  do
+    if [[ $item =~ ^$RELEASE_VER ]]; then ALL_OCP_VERSIONS="$ALL_OCP_VERSIONS $item"
+    else
+      ver_num=$(cut -f2 -d '-' <<< $item)
```

Comment:
I think we can do it in 1 command instead of for loop. After cut we can grep for starts with(^) release version. What do you think?